server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -433,37 +406,6 @@ private void validateGatewayConfig(GatewayConfigDocument configDoc, List<String>
     }
 
     /**
-     * Write out an input stream to a new file.
-     *
-     * @param in       the input stream
-     * @param fileName the file name
-     * @return whether file was successfully written
-     * @throws IOException
-     */
-    private void writeToNewFile(InputStream in, String fileName) throws IOException {
```

Comment:
In the past we output the new config .xml as gateway-config.xml.new or something like this.  (4.x line),  I'm assuming by the removal of this method we no longer do that?  I suggest that we should do that... thoughts?